pkg/descriptions.json

```diff
@@ -26,6 +26,11 @@
     ]
   },
   {
+    "Name": "Microsoft.Diagnostics.Tracing.EventSource.Redist",
+    "Description": "Microsoft.Diagnostics.Tracing.EventSource NuGet package which provides an enhanced developer experience.",
```

Comment:
Lame description.  This is going to appear in the nuspec and in the gallery.  You probably want the same text you already have here: https://www.nuget.org/packages/Microsoft.Diagnostics.Tracing.EventSource.Redist/.

Also consider adding common types, this will also appear in the gallery / package description so it helps for folks searching for your package.

Reply:
Fixed.  Thanks.